app/lib/isClickWithinStaveBounds.ts

```diff
@@ -1,21 +1,38 @@
 import { StaveType } from "./typesAndInterfaces";
+
 const isClickWithinStaveBounds = (
```

Comment:
Did refactoring the isClickWithinStaveBounds cause the out of bounds bug we saw the other day? Or was that already fixed?